src/resources/base.js

```diff
@@ -1,5 +1,5 @@
-import ValidationError from '../errors/validation';
 import NotImplementedError from '../errors/not_implemented';
```

Comment:
# Reviewer Note

Most of the changes in this file were simply organization and did not adjust logic as it was previously near impossible to reason about what things were where as there were similar functions spread all over this file.